app/assets/javascripts/show_errors.js

```diff
@@ -37,14 +37,23 @@ let ErrorDisplay = {
   hideError(e) {
     $(e.target).removeClass('has-error').parent().removeClass('has-error');
     $(e.target).siblings('.error-msg').remove();
+    $(e.target).parent('.error-msg').remove();
   },
 
   findField(field_name, $form, response) {
     if (response.name) {
       field_name = [response.name, '[', field_name, ']'].join('');
     }
-    return $form.find("[name='" + field_name + "']");
+    var field = $form.find("[name='" + field_name + "']");
```

Comment:
You just changed the beginning of this line, but I wish I'd written this code using ES6 string interpolation as 
```
$form.find(`[name="${field_name}"]`)
```